<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerWindow</class>
 <widget class="QMainWindow" name="ViewerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>724</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QWidget" name="CameraControlWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>420</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="CameraListWidget" name="cameraListWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ParaSettingsWidget" name="paraSettingWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="RenderWindow" name="renderWindow" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuAutoNameWhenCapturuing">
     <property name="title">
      <string>Auto file naming</string>
     </property>
     <addaction name="actionOn"/>
     <addaction name="actionOff"/>
    </widget>
    <addaction name="actionsetDefaultSaveDir"/>
    <addaction name="menuAutoNameWhenCapturuing"/>
    <addaction name="actionplayFile"/>
    <addaction name="actionConvertDepth2PC"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionChinese"/>
    <addaction name="actionEnglish"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionGithub"/>
    <addaction name="actionWebSite"/>
    <addaction name="actionOpenLogDir"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionRestartCamera"/>
    <addaction name="actionInfomation"/>
    <addaction name="actionIpSetting"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Windows</string>
    </property>
    <widget class="CSMenu" name="menuLayout">
     <property name="title">
      <string>Layout</string>
     </property>
     <addaction name="actionTile"/>
     <addaction name="actionTabs"/>
    </widget>
    <widget class="CSMenu" name="menuViews">
     <property name="title">
      <string>Views</string>
     </property>
    </widget>
    <addaction name="menuLayout"/>
    <addaction name="menuViews"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCamera"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>appExit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionChinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionWebSite">
   <property name="text">
    <string>Web Site</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionInfomation">
   <property name="text">
    <string>Information</string>
   </property>
  </action>
  <action name="actionRestartCamera">
   <property name="text">
    <string>Restart camera</string>
   </property>
  </action>
  <action name="actionOpenLogDir">
   <property name="text">
    <string>Open log directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionsetDefaultSaveDir"/>
  <action name="actionGithub">
   <property name="text">
    <string>Github</string>
   </property>
  </action>
  <action name="actionIpSetting">
   <property name="text">
    <string>IP setting</string>
   </property>
  </action>
  <action name="actionplayFile">
   <property name="text">
    <string>Playback</string>
   </property>
  </action>
  <action name="actionConvertDepth2PC">
   <property name="text">
    <string>Convert depth data to point cloud</string>
   </property>
  </action>
  <action name="actionOn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>On</string>
   </property>
  </action>
  <action name="actionOff">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="actionTabs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tabs</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RenderWindow</class>
   <extends>QWidget</extends>
   <header>renderwindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CameraListWidget</class>
   <extends>QWidget</extends>
   <header>cameralistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ParaSettingsWidget</class>
   <extends>QWidget</extends>
   <header>parasettingswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CSMenu</class>
   <extends>QMenu</extends>
   <header>csmenu.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
